python/contourTreeAlignment.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python
+
+from paraview.simple import *
+
+# create a new 'TTK CinemaReader'
+ttkCinemaReader = TTKCinemaReader(DatabasePath='./heatedCylinder/heatedCylinder_2D_raw.cdb')
+
+# create a new 'TTK CinemaQuery'
+ttkCinemaQuery = TTKCinemaQuery(InputTable=ttkCinemaReader)
+ttkCinemaQuery.SQLStatement = """SELECT * FROM InputTable0 WHERE time=4.2"""
+
+# create a new 'TTK CinemaProductReader'
+ttkCinemaProductReader = TTKCinemaProductReader(Input=ttkCinemaQuery)
+
+# create a new 'TTK TopologicalSimplificationByPersistence'
+ttkTopologicalSimplificationByPersistence = TTKTopologicalSimplificationByPersistence(registrationName='TTKTopologicalSimplificationByPersistence', Input=ttkCinemaProductReader)
+ttkTopologicalSimplificationByPersistence.InputArray = ['POINTS', 'nrrd']
+ttkTopologicalSimplificationByPersistence.PersistenceThreshold = 0.05
+
+# create a new 'TTK Merge and Contour Tree (FTM)'
+ttkMergeandContourTreeFTM = TTKMergeandContourTreeFTM(registrationName='TTKMergeandContourTreeFTM', Input=ttkTopologicalSimplificationByPersistence)
+ttkMergeandContourTreeFTM.ScalarField = ['POINTS', 'nrrd']
+ttkMergeandContourTreeFTM.InputOffsetField = ['POINTS', 'nrrd']
```

Comment:
I don't believe this line is necessary.
Thanks for double checking (e.g. with diff or paraview) that the outputs of the script are unchanged after the edit.